src/migrations/00003-add-release-build-log.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "release" ADD COLUMN "build log" TEXT NULL;
```

Comment:
It's good to be defensive and only attempt to add the column if it does not already exit, i.e. `ALTER TABLE "release" ADD COLUMN IF NOT EXISTS "build log" TEXT NULL;`

Reply:
@dfunckt Makes sense, updated.